### ISSUE TITLE
Firmware version

### ISSUE THREAD

**Issue by bartek810** [mouldybread/DCS-6100LH#1]
Hey. So is the highest firmware for RTSP 1.01?

**Comment by fightforlife** [comment]
I just bought this camera today and just connected it to power.
In Accesspoint Mode without reset or downgrade the rtsp streams are available here. I do not know whats the difference:
rtsp://admin:PIN@192.168.0.20:554/live/profile.0/video
rtsp://admin:PIN@192.168.0.20:554/live/profile.1/video

**Comment by mouldybread** [comment]
What version is your firmware, @fightforlife?

**Comment by bartek810** [comment]
Just updated to 1.04.05 RTSP working well.

**Comment by gurkburk76** [comment]
I can confirm that rtsp is working with both "profiles" (or rather urls) with firmware 1.02.02(3.5.16-b06) that came with my cameras.
It would be a good idea to block these cameras from reaching internet by fiddling in your firewall or however you want to do it, never trust d-link as we can't rule out the force-feed you an update or something ;)

This seems to have a small downside consisting in that the red/orange light on the cam keeps blinking as it's not able to reach dlink servers. Some black electrical tape over the led should fix that :)

**Comment by ErikAndren** [comment]
> I can confirm that rtsp is working with both "profiles" (or rather urls) with firmware 1.02.02(3.5.16-b06) that came with my cameras. It would be a good idea to block these cameras from reaching internet by fiddling in your firewall or however you want to do it, never trust d-link as we can't rule out the force-feed you an update or something ;)
> 
> This seems to have a small downside consisting in that the red/orange light on the cam keeps blinking as it's not able to reach dlink servers. Some black electrical tape over the led should fix that :)

Interesting. I blocked all internet access after setup and have no issues with the LED blinking.

**Comment by fightforlife** [comment]
For me it is the same as gurkburk76. Also the timestamp will always start at 01.01.2000 after reboot.

**Comment by waqaslam** [comment]
> rtsp://admin:PIN@192.168.0.20:554/live/profile.0/video

I just updated my DCS-6100LH to the latest firmware **1.04.05(3.5.23-b01)** and unfortunately I've lost access to rtsp streams for both profile 0 and 1.
Before when it wasn't updated, I was successfully able to access rtsp streams in VLC, android's tinyCamPro and motioneye systems but this is not the case anymore.

Can you please write the URL you are accessing to access the rtsp stream?

**Comment by propyless** [comment]
I have the same issue as  @waqaslam. I updated to 1.04.05, none of the RTSP urls are working and recovery webpage to invoke the downgrade does not exist. I did a quick nmap and it said the http port is now on 8080, but I think the webserver ignores all requests that don't match a specific path since just going to the  192.168.0.20:8080/ does not work.

**Comment by ErikAndren** [comment]
I have upgraded to 1.04.05 and can still access the profile0 stream. profile1 has been converted to some kind of echo show stream.

The recovery web page is on port 80. 
You need to press the reset button for at least 5 seconds to enter recovery mode.
The led indicator should change color to red.

**Comment by ErikAndren** [comment]
> > Just updated to 1.04.05 RTSP working well. Edit just profile0 is working. And also you can turn off auto update in the app
> 
> I just updated my DCS-6100LH to the latest firmware 1.04.05(3.5.23-b01) and unfortunately I've lost access to rtsp streams for both profile 0 and 1. Before when it wasn't updated, I was successfully able to access rtsp streams in VLC, android's tinyCamPro and motioneye systems but this is not the case anymore.
> 
> Can you please write the URL you are accessing to access the rtsp stream?

This is the url I am using in vlc. Obviously you need to change the ip-address:
rtsp://192.168.20.11:554/live/profile.0/video

**Comment by propyless** [comment]
> I have upgraded to 1.04.05 and can still access the profile0 stream. profile1 has been converted to some kind of echo show stream.
> 
> The recovery web page is on port 80. You need to press the reset button for at least 5 seconds to enter recovery mode. The led indicator should change color to red.

Yep, my bad, I think I pushed the reset button too early in the boot phase. In my defense the orange and red look super similar...

I now have the rtsp stream accessible.

Thanks @ErikAndren

**Comment by deployHuman** [comment]
> I can confirm that rtsp is working with both "profiles" (or rather urls) with firmware 1.02.02(3.5.16-b06) that came with my cameras. It would be a good idea to block these cameras from reaching internet by fiddling in your firewall or however you want to do it, never trust d-link as we can't rule out the force-feed you an update or something ;)
> 
> This seems to have a small downside consisting in that the red/orange light on the cam keeps blinking as it's not able to reach dlink servers. Some black electrical tape over the led should fix that :)

yeah, my dcs got force-fed an auto update last night. it went up to 1.04.05.
going to need firewall blocking on this i think

**Comment by ErikAndren** [comment]
> > I can confirm that rtsp is working with both "profiles" (or rather urls) with firmware 1.02.02(3.5.16-b06) that came with my cameras. It would be a good idea to block these cameras from reaching internet by fiddling in your firewall or however you want to do it, never trust d-link as we can't rule out the force-feed you an update or something ;)
> > This seems to have a small downside consisting in that the red/orange light on the cam keeps blinking as it's not able to reach dlink servers. Some black electrical tape over the led should fix that :)
> 
> yeah, my dcs got force-fed an auto update last night. it went up to 1.04.05. going to need firewall blocking on this i think

You can disable the auto-update in the mydlink app.

**Comment by deployHuman** [comment]
> > > I can confirm that rtsp is working with both "profiles" (or rather urls) with firmware 1.02.02(3.5.16-b06) that came with my cameras. It would be a good idea to block these cameras from reaching internet by fiddling in your firewall or however you want to do it, never trust d-link as we can't rule out the force-feed you an update or something ;)
> > > This seems to have a small downside consisting in that the red/orange light on the cam keeps blinking as it's not able to reach dlink servers. Some black electrical tape over the led should fix that :)
> > 
> > 
> > yeah, my dcs got force-fed an auto update last night. it went up to 1.04.05. going to need firewall blocking on this i think
> 
> You can disable the auto-update in the mydlink app.

i think that option is lost eh ?
i cant find anything about auto-update in the mydlink app

**Comment by ErikAndren** [comment]
> > > > I can confirm that rtsp is working with both "profiles" (or rather urls) with firmware 1.02.02(3.5.16-b06) that came with my cameras. It would be a good idea to block these cameras from reaching internet by fiddling in your firewall or however you want to do it, never trust d-link as we can't rule out the force-feed you an update or something ;)
> > > > This seems to have a small downside consisting in that the red/orange light on the cam keeps blinking as it's not able to reach dlink servers. Some black electrical tape over the led should fix that :)
> > > 
> > > 
> > > yeah, my dcs got force-fed an auto update last night. it went up to 1.04.05. going to need firewall blocking on this i think
> > 
> > 
> > You can disable the auto-update in the mydlink app.
> 
> i think that option is lost eh ? i cant find anything about auto-update in the mydlink app

Indeed, this is because you are on the latest version. 
You can force a downgrade by following the instructions in the README of this repository. 
Then you can disable the autoupdate in the app.

**Comment by andrewbeyou88** [comment]
Does anyone still has a firmware file available? i can't find no link available anymore. Any version is good.
Thanks

**Comment by mouldybread** [comment]
It looks like you can still get 1.01.01 from here: https://www.dlinktw.com.tw/techsupport/ProductInfo.aspx?m=DCS-6100LH

The top row is for firmware and 下載 means download

**Comment by andrewbeyou88** [comment]
> It looks like you can still get 1.01.01 from here: https://www.dlinktw.com.tw/techsupport/ProductInfo.aspx?m=DCS-6100LH
> 
> The top row is for firmware and 下載 means download

Thanks for your help!

**Comment by ErikAndren** [comment]
https://github.com/ErikAndren/DCS-6100LH/tree/main/firmwares
